packages/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -5,14 +5,15 @@ All notable changes to this tool will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
 ### Added
 
 -   We added support to the typings generator for the data source caption (`{caption: string}`), introduced in version 9.24.0. This feature will give widget developers the ability to display the same caption for a data source, that Mendix uses in the Page Explorer, within a widget preview.
 
 ### Changed
 
--   We've updated our library versions (React and React Native), please be aware this might break your widgets. New libraries versions: React: 18.0.2, React-Native: 0.70.7. This is necessary due to breaking changes introduced in Studio Pro 9.24.0.
 -   We pinned our version of Typescript to 4.9.5
```

Comment:
🎄Let's change "pinned" to "bumped", spell TypeScript with a capital S and add a `.` at the end of the sentence.

Reply:
It is pinned to a fixed version.

![image](https://user-images.githubusercontent.com/6069943/228164664-5337be0f-a120-4f33-ac1f-0dbe66310b2d.png)

Pinned would be more accurate to say? What you think?

Reply:
Dot added to the end

Reply:
Fixed ✨